File: AASC.Partner/Models/PhaseOut/PhaseOutPrepViewModel.cs
using System;
using System.Collections.Generic;

namespace AASC.Partner.API.Models
{
    public class PhaseOutPrepViewModel
    {
        public string Id { get; set; }

        public bool Phased { get; set; }
        
        public string PartNumber { get; set; }
        
        public string Description { get; set; }
        
        public string PlmStatus { get; set; }
        
        public string ProductFamily { get; set; }
        
        public string LastBuyTime { get; set; }
        
        public string Replacement { get; set; }

        public string CreatedBy { get; set; }
        
        public string CreatedDate { get; set; }
    }

}

File: AASC.Partner.Web/AASC.Partner.Web/app/viewModels/phaseout/phaseOutPrepViewModel.js
'use strict';

app.factory('phaseOutPrepViewModel', function () {
    return kendo.data.Model.define({
        id: 'id',
        fields: {
            id: { type: 'string', editable: false, nullable: false },
            phased: { type: 'boolean', editable: true, nullable: false },
            partNumber: { type: 'string', editable: true, nullable: false },
            description: { type: 'string', editable: true, nullable: true },
            plmStatus: { type: 'string', editable: true, nullable: false },
            productFamily: { type: 'string', editable: true, nullable: false },
            lastBuyTime: { type: 'date', editable: true, nullable: true },
            replacement: { type: 'string', editable: true, nullable: true },
            createdBy: { type: 'string', editable: true, nullable: false },
            createdDate: { type: 'date', editable: true, nullable: false }
        }
    });
});